schedule/virt_autotest/sle16_guest_installation.yaml

```diff
@@ -0,0 +1,9 @@
+name:          sle16-virt-guest-installation
+description:    >
+    Virtualization guest installation tests including extended feature tests schedule with agama and IPXE
+vars:
+schedule:
+    - installation/ipxe_install
+    - installation/agama_reboot
+    - virt_autotest/login_console
+    - virt_autotest/unified_guest_installation
```

Comment:
Does unified_guest_installation cover any code paths that you would like to cover atm? If not, maybe removing it will let the host installation stop at a proper step and finish soon enough.

Reply:
It is just for helping guest installation and feature tests since it is in dev group. Without `unified_guest_installation`, we can consider of moving the test suite to the formal product job group.

Reply:
Why not ;-) ? I suggest to have some official test even if it is short path now, better than empty.

Reply:
Ok. let me do it now.

Reply:
done. and new verification job: https://openqa.suse.de/tests/16444961